### PR TITLE
Bugfix/metrics-warning

### PR DESCRIPTION
**Pull request by fivetran-sheringuyen**
**Are you a current Fivetran customer?** 

Fivetran Employee 🥨 

**What change(s) does this PR introduce?** 

Updates to `ad_reporting_metrics.yml` in order to stop warnings when running package.

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)
It currently looks like this update should work for both dbt-core 1.3+ according to dbt docs.

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue/Feature [#74 ]
- [ ] No 

**How did you test the PR changes?** 

 

- [x] Buildkite  
- [x] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [x] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🍳 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

**Comment by fivetran-sheringuyen** [comment]
Docs were not updated because I don't _believe_ this change should affect the current docs. But happy to update if that is false.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
@fivetran-sheringuyen I just tested this on both 1.4 and 1.3 dbt core versions and it works as expected with the warning removed! My only request/question is that we should update the metrics version note in the README (see below)
image

This should be greater than 1.3.0 I believe. Additionally, is there anything we need to add for version requirements with the dbt metrics package?

**Comment by fivetran-joemarkiewicz** [comment]
@fivetran-sheringuyen before we release this, let's make sure we integrate PR #81 into the v0.2.1 release. I would rather batch these updates than do back to back minor releases.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
+# dbt_ad_reporting v1.2.1
+
+## Updates
+- Updating `ad_reporting_metrics.yml` to be up to date with [dbt Metrics documentation](https://docs.getdbt.com/docs/build/metrics#derived-metrics) ([PR #82](https://github.com/fivetran/dbt_ad_reporting/pull/82))
+
+## Bug Fixes
+- Enabling additional Snapchat Ads columns in `ad_reporting__url_report` that were previously mapped to null values. ([#81](https://github.com/fivetran/dbt_ad_reporting/pull/81))
+  - These columns are: ad_group_id (ad_squad_id), ad_group_name (ad_squad_name), campaign_id and campaign_name.
```
Comment (fivetran-reneeli) [inline CHANGELOG.md:8]: I would just instead note that campaign id and name were dropped from the mapping

**Pull request merged**